### PR TITLE
Policy-Check: Fix RegExp hang/DOS

### PR DESCRIPTION
**Pull request by DLehenbauer**
PR #5767 contains generated *.d.ts files that trigger a deterministic RegExp based hang/denial-of-service when vetting copyright headers.

Example 'policy-check' run that exceeded 60 min:
https://github.com/microsoft/FluidFramework/pull/5767/checks?check_run_id=2355639340

This change fixes the issue by requiring an exact match that begins at the first line of the file.

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,5 @@
-#-------------------------------------------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
-#-------------------------------------------------------------------------------------------------------------
+# Licensed under the MIT License.
```
Comment (DLehenbauer) [inline .devcontainer/Dockerfile:3]: The new copyright scanner is stricter.  (This file was one of two Dockerfiles that had a 'unique' header.)

File: examples/data-objects/webflow/src/test/flowdocument.spec.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /*!
-* Copyright (c) Microsoft Corporation. All rights reserved.
-* Licensed under the MIT License.
-*/
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
```
Comment (DLehenbauer) [inline examples/data-objects/webflow/src/test/flowdocument.spec.ts:5]: Spaces required before '*'

File: examples/hosts/app-integration/external-controller/tests/index.ts [hunk 3]
```diff
@@ -1,9 +1,10 @@
-/* eslint-disable import/no-internal-modules */
 /*!
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
 
+/* eslint-disable import/no-internal-modules */
+
 import { KeyValueDataObject } from "@fluid-experimental/data-objects";
```
Comment (DLehenbauer) [inline examples/hosts/app-integration/external-controller/tests/index.ts:8]: Header must appear as first line.

File: experimental/dds/tree-graphql/src/graphql-generated/Pizza.ts [hunk 4]
```diff
@@ -1,3 +1,8 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
 export type Maybe<T> = T | null;
```
Comment (DLehenbauer) [inline experimental/dds/tree-graphql/src/graphql-generated/Pizza.ts:6]: (not sure how this file slipped by before.)

**Review by SamBroner — APPROVED** [review]
Awesome. And this will make it easier if we ever need to update the copyrightText

**Pull request merged**